index.html

```diff
@@ -6003,6 +5861,113 @@ <h3>RemoteDocument</h3>
     </section>
   </section> <!-- end of Remote Document and Context Retrieval -->
 
+  <section class="changed">
+    <h2>HTML Content Algorithms</h2>
+    <p class="note">This section describes features available
+      with a <a data-link-for="JsonLdOptions">documentLoader</a> supporting HTML script extraction.</p>
+    <p>Implementations of a <a data-link-for="JsonLdOptions">documentLoader</a> MAY support extracting JSON-LD from
+      <a data-cite="HTML/scripting.html#the-script-element">script elements</a> contained within an HTML [[HTML]] document.
+      This section describes the normative behavior of such processors.
+      Such a processor supports <dfn>HTML script extraction</dfn>.</p>
+
+    <section id="process-html"><h3>Process HTML</h3>
+      <p>This sections describe an extension to the algorithm specified
+        in <a>LoadDocumentCallback</a> to support extracting JSON-LD from HTML.</p>
+
+      <p><a href="#LoadDocumentCallback-step-2">Step 2</a> is updated to add the following: A processor supporting <a>HTML script extraction</a> MUST include <code>text/html</code> at any preference level,
+        unless <a data-link-for="LoadDocumentOptions">requestProfile</a> is `http://www.w3.org/ns/json-ld#context`.</p>
+
+      <p>After <a href="#LoadDocumentCallback-step-5">step 5</a>, add the following processing step:
+        Otherwise, if the retrieved resource's <a>Content-Type</a> is <code>text/html</code>:</p>
+      <ol>
+        <li>If the processor does not support <a>HTML script extraction</a>
```

Comment:
It is a bit strange to have this test. As I understand this section, the algorithm should be so updated IF the document loader supports HTML script extraction. So the test would be trivially true...

Reply:
Yes, that step is not necessary.